packages/chrome/src/elements/sheet/Sheet.tsx

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useRef,
+  useMemo,
+  useState,
+  HTMLAttributes,
+  PropsWithoutRef,
+  RefAttributes,
+  ForwardRefExoticComponent
+} from 'react';
+import PropTypes from 'prop-types';
+import { useUIDSeed } from 'react-uid';
+import mergeRefs from 'react-merge-refs';
+
+import { StyledSheet, StyledSheetWrapper } from '../../styled';
+import { SheetContext } from '../../utils/useSheetContext';
+import { useFocusableMount } from '../../utils/useFocusableMount';
+
+import { SheetTitle } from './components/Title';
+import { SheetDescription } from './components/Description';
+import { SheetHeader } from './components/Header';
+import { SheetBody } from './components/Body';
+import { SheetFooter } from './components/Footer';
+import { SheetFooterItem } from './components/FooterItem';
+import { SheetCloseButton } from './components/Close';
+
+interface IStaticSheetExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Title: typeof SheetTitle;
+  Description: typeof SheetDescription;
+  Header: typeof SheetHeader;
+  Body: typeof SheetBody;
+  Footer: typeof SheetFooter;
+  FooterItem: typeof SheetFooterItem;
+  Close: typeof SheetCloseButton;
+}
+
+export interface ISheetProps extends HTMLAttributes<HTMLElement> {
+  /** Sets the root ID. A unique ID is created if none is provided. **/
+  id?: string;
```

Comment:
Please note that `id`, which naturally exists on the extended element, is never documented throughout the Garden codebase. Reference container `useUIDSeed` usages for Tabs, Accordion, form Field, Modal, Tooltip. This can be removed.

Reply:
Makes sense, I will remove this.